python/src/GaussianProcessConditionalCovariance_doc.i.in

```diff
@@ -147,27 +156,33 @@ Parameters
 x : sequence of float
     The point :math:`\vect{x}` where the conditional variance of the output has to be evaluated.
 sampleX : 2-d sequence of float
-     The sample :math:`(\vect{\xi}_1, \dots, \vect{\xi}_M)` where the conditional variance of the output has to be evaluated (*M* can be equal to 1).
+     The sample :math:`(\vect{\xi}_1, \dots, \vect{\xi}_N)` where the conditional variance of the output has to be evaluated (*N* can be equal to 1).
 marginalIndex : int
     Marginal of interest (for multiple outputs).
-    Default value is 0
+    
+    Default value is 0 (first component).
 marginalIndices : sequence of int
     Marginals of interest (for multiple outputs).
 
 Returns
 -------
 var : float
-      Variance of interest.
-      float if one point (x) and one marginal of interest (x, marginalIndex)
+      The variance of the specified marginal of the Gaussian process regression  :math:`\vect{Z}`
+      defined in :eq:`GPRdef` at the specified point.
 
 varPoint : sequence of float
-    The marginal variances
+    The marginal variances of each marginal of interest computed at each given point.
 
 
 Notes
 -----
-In case of fourth usage, the sequence of float is given as the concatenation of marginal variances 
-for each point in sampleX."
+If only one  marginal :math:`k`  of interest  and one point  :math:`\vect{x}` have been specified,
+the method returns :math:`\Var{Z_k(\omega, \vect{x}}` where :math:`\vect{Z}` is the Gaussian
+process regression  defined in :eq:`GPRdef`.
+
+If several marginal of interest :math:`k_1, \dots, k_M)` or several points :math:`(\vect{\xi}_1,
+\dots, \vect{\xi}_N)` have been specified, the method returns the concatenation of
+sequence of variances :math:`(\Var{Z_{k_1}(\omega, \vect{\xi}_j}, \dots, \Var{Z_{k_M}(\omega, \vect{\xi}_j})` for each :math:`1 \leq j \leq N`."
```

Comment:
```suggestion
sequence of variances :math:`(\Var{Z_{k_1}(\omega, \vect{\xi}_j}), \dots, \Var{Z_{k_M}(\omega, \vect{\xi}_j}))` for each :math:`1 \leq j \leq \sampleSize`."
```

Reply:
ok for the missging ) but I keep the notation N rather than \sampleSize because this size has no link with the size of the dataset used to create the GPR.